t/config.t

```diff
@@ -95,7 +95,7 @@ subtest 'Test configuration default modes' => sub {
     # Test configuration generation with an unknown mode (should fallback to default)
     $app = Mojolicious->new();
     $app->mode("foo_bar");
-    OpenQA::ServerStartup::read_config($app);
+    OpenQA::FakeApp::read_config($app);
```

Comment:
Can we not spread the name FakeApp then?

Reply:
...what about an OpenQA Base class then? :)